mailer/engine.py

```diff
@@ -112,6 +113,24 @@ def release_lock(lock):
     logging.debug("released.")
 
 
+def error_handler(connection, message, err):
```

Comment:
This definitely needs a doc-string explaining the interface, return value structure and valid return values.

Reply:
I would call the function `handle_backend_exception`, instead.